templates/rhpam72-prod.yaml

```diff
@@ -251,22 +256,22 @@ parameters:
   name: KIE_SERVER2_USE_SECURE_ROUTE_NAME
   value: 'false'
   required: false
-- displayName: Business Central Monitoring Server Keystore Secret Name
+- displayName: Business Central Monitoring Server and Smart Router Keystore Secret Name
   description: The name of the secret containing the keystore file
   name: BUSINESS_CENTRAL_HTTPS_SECRET
   example: businesscentral-app-secret
   required: true
-- displayName: Business Central Monitoring Server Keystore Filename
+- displayName: Business Central Monitoring Server and Smart Router Keystore Filename
```

Comment:
Wouldn't it have more sense to split Smart router and Business central keystore configuration?

Reply:
The rational here was backward compatibility, since a secret was not required for the router in previous versions of the template. Since the business central secret was already required, this gets us https for the smart router without adding necessary template params.

Reply:
having said that, if we think backwards compat is not an issue in this case, I'm happy to add another secret

Reply:
`@tmckayus I'm sorry, I know we had spoken about this before, but the more I think about it, I think @sutaakar is right that it's the "more correct" thing to do. Given that, can you separate the smartrouter from the businesscentral https config, so that they don't share variables/values? Thank you!

Reply:
@errantepiphany absolutely. Are there associated docs to modify for sample template usage?

Reply:
@tmckayus The adocs will get regenerated based on the templates before release.